Makefile

```diff
@@ -1,6 +1,7 @@
 NAME          := hdf5
-SRC_EXT       := bz2
+SRC_EXT       := gz
 TEST_PACKAGES := $(NAME) java-$(NAME) $(NAME)-devel $(NAME)-static $(NAME)-mpich-tests $(NAME)-openmpi3-tests
+DL_VERSION    := 5b5a1a81029eb7bdc975beff5f18c9c440f5de56
```

Comment:
The delta between the nearest _release_ from upstream and this git hash should be a patch in this repo, not a URL to download the release from.  For transparency about what is actually being built/packaged.

Reply:
The latest release was 1.12 and that release is currenly 1300+ commits behind the above commit.

Reply:
That's exactly the kind of transparency we are looking to exhibit.  When somebody looks at our SRPM, they should see that we have applied a huge patch to the latest release rather than that being buried in pulling some anonymous commit.

That also should be prompting us to prod upstream for a new release.  Are those 1300+ commits really so small and insignificant that they don't justify a new release?  Maybe.  Maybe not.

Reply:
sorry but transparency to who? we are not distributing any of this and it's only for our CI testing, and the idea of making this giant patch seems wrong to me. i think we should switch back to the commit hash.